components/widgets/pulsating_dot/pulsating_dot.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
We can rename this file to `index.tsx` to avoid the extra `index` file in this folder that just imports and exports